### DISCUSSION TITLE
Igraph Install for Ubuntu User

### DISCUSSION THREAD

**Discussion by anthonyjlau** [sailuh/kaiaulu#270] [Q&A] [answered]
When I tried to download the Igraph package from RStudio, I got an error and the package did not install.

**Comment by anthonyjlau** [comment, marked as answer]
In order to solve this I found a stack overflow forum explaining how to download it from the terminal. The top answer is the strategy I used to get igraph installed.  [Installation of igraph package of R in ubuntu](https://stackoverflow.com/questions/40995888/installation-of-igraph-package-of-r-in-ubuntu).

**Comment by szhorvat** [comment]
Installation instructions are in the README on GitHub:

https://github.com/igraph/rigraph/?tab=readme-ov-file#installation
